autoload/lsp/completion.vim

```diff
@@ -641,7 +641,7 @@ export def BufferInit(lspserver: dict<any>, bnr: number, ftype: string)
     return
   endif
 
-  if !opt.lspOptions.autoComplete && !LspOmniComplEnabled(ftype)
+  if !opt.lspOptions.autoComplete && !LspOmniComplEnabled(ftype) && !opt.lspOptions.omniComplete
```

Comment:
tho not sure your problem, but the code logic seems wrong:
- when `lspOptions.omniComplete` was true, 
- but `server.omnicompl` was false,
- then it should be still no omni compl there.
a.k.a when auto compl was false too, then that judgement should be `return` still.

Reply:
a.k.a when auto compl was false too, then ~~that judgement should be return still.~~ it already like its next comment said.